src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Extensions/ReleaseAmendmentExtensionsTests.cs

```diff
@@ -515,6 +515,64 @@ public void CreateAmendment_ClonesContentBlocks_SameAsBlocksInContent()
             Assert.Equal(block2, contentSection1Block2);
         }
 
+        [Fact]
+        public void CreateAmendment_CopiesKeyStatistics()
+        {
+            var dataBlock = new DataBlock();
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                Version = 1,
+                Published = DateTime.Parse("2020-10-10T13:00:00"),
+                PublishScheduled = DateTime.Parse("2020-10-09T12:00:00"),
+                KeyStatistics = new List<KeyStatistic>
+                {
+                    new KeyStatisticText { Id = Guid.NewGuid() },
+                    new KeyStatisticDataBlock
+                    {
+                        Id = Guid.NewGuid(),
+                        DataBlock = dataBlock,
+                    },
+                },
+                ContentBlocks = new List<ReleaseContentBlock>
+                {
+                    new ReleaseContentBlock
+                    {
+                        ContentBlock = dataBlock,
+                    },
+                },
+            };
+
+            var createdDate = DateTime.Now;
+            var createdById = Guid.NewGuid();
+
+            var amendment = release.CreateAmendment(createdDate, createdById);
+
+            Assert.NotEqual(release.Id, amendment.Id);
+            Assert.Equal(2, amendment.Version);
+            Assert.Equal(release.Id, amendment.PreviousVersionId);
+
+            Assert.Null(amendment.Published);
+            Assert.Null(amendment.PublishScheduled);
+            Assert.Equal(ReleaseApprovalStatus.Draft, amendment.ApprovalStatus);
+
+            Assert.Equal(2, amendment.KeyStatistics.Count);
+
+            var amendmentKeyStatText = Assert.IsType<KeyStatisticText>(amendment.KeyStatistics[0]);
+            Assert.NotEqual(release.KeyStatistics[0].Id, amendmentKeyStatText.Id);
+
+            var amendmentKeyStatDataBlock = Assert.IsType<KeyStatisticDataBlock>(amendment.KeyStatistics[1]);
```

Comment:
What I wanna know with this test is, does the amendment KeyStatisticDataBlock have the same DataBlockId as the original, or do we expect the DataBlock to be copied and thus have a different DataBlockId?  Worth adding this to the test, unless it's already here and I just can't see it?

Reply:
The data block is copied and the copy has a new `Id`.

I think I was already testing this in the code immediately below, but it wasn't particularly clear. I've updated the test so hopefully now it's is clearer. Happy to make further changes if you don't think it's clear or I've not addressed your point.

Reply:
Nice that's cool thanks